lib/modules/actions.js

```diff
@@ -54,7 +56,7 @@ export const reroutePage = () => async (dispatch, getState) => {
       queryParams: currentPage.queryParams,
       hashParams: currentPage.hashParams,
       bodyParams: {},
-      referrer: currentPage.referrer,
+      referrer: currentPage.referrer || NO_REFERRER,
```

Comment:
The set of all possible referrers includes the null value. I am not in favor of passing a constant when a null value would be just fine. If you want a defined fallback, I would prefer `undefined`.